docs/storage.rst

```diff
@@ -82,4 +82,24 @@ Example settings
                 },
             },
         },
+        "backup": {
```

Comment:
It doesn't make sense to have this documentation here.

Reply:
Should I move it to genialis base?